### ISSUE TITLE
fieldset with textbox elements: reopen issue #7283

### ISSUE THREAD

**Issue by biljanap** [wet-boew/wet-boew#7936]
I would like to reopen #7283 because I have the exact same situation again.

**Comment by kodecount** [comment]
The default method in jQuery Validate is:
```javascript
<div class="wb-frmvld">
<form action="#" method="get" id="validation-example">
<fieldset class="phone-group">
<legend class="required"><span class="field-name">Phone Book</span> <strong class="required">(required)</strong></legend>
<div class="form-group">
<label for="phone1">Cell Phone</label>
<input class="form-control" id="phone" name="phone1" type="text" data-rule-require_from_group='[1, ".phone-group"]' />
</div>
<div class="form-group">
<label for="phone2">Home Phone</label>
<input class="form-control" id="phone" name="phone2" type="text" data-rule-require_from_group='[1, ".phone-group"]' />
</div>
<div class="form-group">
<label for="phone3">Work Phone</label>
<input class="form-control" id="phone3" name="phone3" type="text" data-rule-require_from_group='[1, ".phone-group"]' />
</div>
</fieldset>
<input type="submit" value="Submit" class="btn btn-primary" /> <input type="reset" value="Reset" class="btn btn-default" />
</form>
</div>
```
The only issue is the error message doesn't display properly (displays on the last input).

**Comment by RobJohnston** [comment]
The require_from_group method has had spotty success in WET.  In some versions it works, and it others, it doesn't.  See https://github.com/wet-boew/wet-boew/issues/7067.

**Comment by kodecount** [comment]
Yes and even custom validation for this is tricky since you can't latch onto the error placement call (the error placement call is how WET places the errors on the page).

**Comment by kodecount** [comment]
Did more testing and I can get it to work with WET but before I put a fix in WET the method needs to be fixed in jQuery Validation: https://github.com/jquery-validation/jquery-validation/issues/1972

**Comment by kodecount** [comment]
Here is the fix for WET:
```
var group = $element.hasClass( "require_from_group" ) || $element.attr( "data-rule-require_from_group" );
if ( group ) {
$fieldset = $element.closest( "fieldset" );
if ( $fieldset.length !== 0 ) {
$legend = $fieldset.find( "legend" ).first();
if ( $legend.length !== 0 && $fieldset.find( "input[name='" + $element.attr( "name" ) + "']" ) !== 1 ) {
var strong = $legend.find( "strong.error" ),
id = $legend.attr( "id" );

if ( strong.length === 0 ) {
if ( !id ) {
id = "required-group-" + idCount;
idCount += 1;
}
$error.attr( "id", "#" + id );
$error.appendTo( $legend );
}
return;
}
}
}
```

**Comment by kodecount** [comment]
I was using the method wrong. I thought the class was the `div` or `fieldset` around a group of inputs, but I miss understood the jQuery Validation docs. The class goes with the input. It works fine, but produces three errors. Seems excessive. I will create a PR to have one error.

**Comment by kodecount** [comment]
Please see PR https://github.com/wet-boew/wet-boew/pull/7940 which should fix this issue.

**Comment by lucas-hay** [comment]
Addressed in #7940
